health-services/project-factory/src/server/utils/campaignUtils.ts

```
@@ -1361,58 +1406,84 @@
+  //       return obj.value; // replace with value (just the boundary code)
+  //     } else {
+  //       return obj;
+  //     }
+  //   });
+  // });
+
+  // Part 2: for rows without boundary code
+  const boundaryDataForWithoutBoundaryCode = withoutBoundaryCode.map((row: any[]) => {
+    let boundaryName: string | undefined;
+
+    for (let level of hierarchy) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'level' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)